app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1109,7 +1100,7 @@ public void onClick(View view) {
      */
     private void showSoftKeyboard() {
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
-        imm.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT,InputMethodManager.HIDE_IMPLICIT_ONLY);
+        imm.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT, InputMethodManager.HIDE_IMPLICIT_ONLY);
```

Comment:
I think its probably fine to retain code refactor changes. These occur clearly because someone forgot to refactor code and it was merged. I think it'd be reasonable to include such changes in a different commit :).